### PR TITLE
serializationFormat slingxml with xml output for Sling-Initial-Content

### PR DESCRIPTION
**Pull request by mrozati**
add serializationFormat `slingxml` which can be used for XML outputs for Sling Initial Content. This generates the XML file with the clientlib name.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.5.0
```
Comment (stefanseifert) [inline CHANGELOG.md]: uh, the changelog is outdated, it does not include the latest changes from 1.4.4, 1.5.0 and post-1.5.0.
next version will be 1.6.0

**Comment by stefanseifert** [comment]
@ifahrentholz please review

**Comment by mbehzad** [comment]
@mrozati , re: failed unit-test
the new `getXmlOutputFile` method does not return the `contentXml` value (from clientlib.js:256)
and one of the indentations for the xml output (clientlib.j:205) has one space too much (compared to the test snapshot and other lines in the xml output (:209, :211)

File: CHANGELOG.md (hunk 1 above)
Comment (mrozati) [reply CHANGELOG.md]: oh, I committed my changed on top of master branch, but I just see now that there is also a develop branch, which is "behind" the master.  looks that something is wrong here.

Comment (mrozati) [reply CHANGELOG.md]: oh, I committed my changed on top of master branch, but I just see now that there is also a develop branch, which is "behind" the master.  looks that something is wrong here.

**Comment by mrozati** [comment]
thanks. I fixed them

**Comment by ifahrentholz** [comment]
@stefanseifert: On my machine the tests ran without an issue.

**Pull request merged**